docs/PORTABLE_SERVICES.md

```diff
@@ -6,10 +6,10 @@ layout: default
 
 # Portable Services Introduction
 
-This systemd version includes a preview of the "portable service"
-concept. "Portable Services" are supposed to be an incremental improvement over
-traditional system services, making two specific facets of container management
-available to system services more readily. Specifically:
+systemd since version v239 includes a new "portable service" concept.
```

Comment:
This first sentence is still awkward. Why not just `systemd (since version 239) supports a concept of "portable services":`